evaluations/yieldscan_1_mmagician.md

```diff
@@ -0,0 +1,44 @@
+# Evaluation
+
+* **Status:** In progress
+* **Application Document:** https://github.com/w3f/Grants-Program/blob/master/applications/yieldscan_phase_2.md
+* **Milestone:** 1
+* **Kusama Identity:** [H9M22FTqs1kKRAUiEk1BCVNQfWdMzVV3HbHuMwkirirxk85](https://polkascan.io/pre/kusama/account/H9M22FTqs1kKRAUiEk1BCVNQfWdMzVV3HbHuMwkirirxk85)
+* **Previously successfully merged evaluation:** All by mmagician
+
+| Number | Deliverable                | Accepted               | Link                                                                                                                                                                                                                                                                                                                                                                                 |                                                                                                                                                               |
+|--------|----------------------------|------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------|
+|    0a. | License                    | <ul><li>[x] </li></ul> | <ul><li>[yieldscan-frontend](https://github.com/yieldscan/yieldscan-frontend/blob/master/LICENSE.md)</li><li>[yieldscan-backend-crawler](https://github.com/yieldscan/yieldscan-backend-crawler/blob/master/LICENSE.md)</li><li>[yieldscan-backend-api](https://github.com/yieldscan/yieldscan-backend-api/blob/master/LICENSE.md)</li></ul>                                         |                                                                                                                                                               |
+|    0b. | Documentation              | <ul><li>[ ] </li></ul> | <ul><li>[yieldscan-frontend documentation](https://github.com/yieldscan/yieldscan-frontend/tree/develop#codebase-overview)</li><li>[yieldscan-backend-crawler documentation](https://github.com/yieldscan/yieldscan-backend-crawler#codebase-overview)</li><li>[yieldscan-backend-api documentation](https://github.com/yieldscan/yieldscan-backend-api#codebase-overview)</li></ul> | Documentation regarding the key part of each repo could be expanded, e.g. for the crawler repo, describing in more detail what the crawlers are, what they do |
```

Comment:
as per your advice we have updated our documentation wherever we felt would need more detail for an outside contributor.